include/swift/AST/DiagnosticsSema.def

```diff
@@ -2977,6 +2977,15 @@ WARNING(swift3_ignore_specialized_enum_element_call,none,
         "cannot specialize enum case; ignoring generic argument, "
         "which will be rejected in future version of Swift", ())
 
+WARNING(bound_implicitly_unwrapped_optional_spelling_deprecated,none,
```

Comment:
These should be ERRORs in -swift-version 5 mode.

Reply:
Sure.